packages/graph-explorer/src/modules/EntitiesTabular/components/NodesTabular.tsx

```diff
@@ -70,40 +62,33 @@ const NodesTabular = forwardRef<TabularInstance<ToggleVertex>, any>(
         },
         {
           id: "node-id",
-          accessor: row => textTransform(row.id),
+          accessor: "displayId",
           label: t("entities-tabular.node-id"),
           overflow: "ellipsis",
           oneLine: true,
         },
         {
           id: "node-type",
-          accessor: row =>
-            (row.types ?? [row.type])
-              .map(textTransform)
-              .filter(Boolean)
-              .join(", "),
+          accessor: row => row.displayTypes,
           label: t("entities-tabular.node-type"),
-          filter: (rows, _columnIds, filterValue) => {
-            return rows.filter(row => {
-              const vertex = row.original as Vertex;
-              return (vertex.types ?? [vertex.type]).find(t => {
-                const label = config?.getVertexTypeConfig(t)?.displayLabel || t;
-                return label.toLowerCase().match(filterValue.toLowerCase());
-              });
-            });
-          },
+          filter: (rows, _columnIds, filterValue) =>
+            rows.filter(row =>
+              row.original.displayTypes
```

Comment:
Can `original.displayTypes` have multiple comma separated values here? And if so will it be able to be filtered correctly using `match(filterValue.toLowerCase())`?